doc/sphinx/source/develop/dataset.rst

```diff
@@ -124,15 +169,15 @@ If a single dataset has more than one reference,
 it is possible to add tags as a list e.g. ``reference: ['tag1', 'tag2']``.
 The third part in the configuration file defines the variables that are supposed to be cmorized.
 
-The actual cmorizing script ``cmorize_obs_mte.py`` consists of a header with
+The actual cmorizing script ``mte.py`` consists of a header with
 information on where and how to download the data, and noting the last access
 of the data webpage.
 
 The main body of the CMORizer script must contain a function called
 
 .. code-block:: python
 
-   def cmorization(in_dir, out_dir, cfg, config_user):
+   def cmorization(in_dir, out_dir, cfg, _, __, ___):
```

Comment:
Maybe put here the full signature, which is 
```def cmorization(in_dir, out_dir, cfg, cfg_user, start_date,  end_date):```

Most cmorizers do no use the last three ones, but I think that will become more common in the future

Reply:
Ok, I just did that. Note that none of the scripts uses the 6 arguments at the moment.

Reply:
Could you add a definition for the last 3 arguments? This is currently not in the doc.